frontend/src/components/Histogram.jsx

```diff
@@ -29,48 +29,47 @@ export default class Histogram extends React.Component {
     console.log("Fetching case data");
 
     // if there is not an available start or end date in the data use this default ones
-    const initial_date =
-      this.props.initial_date != null ? this.props.initial_date : "2020-02-20";
-    const maximum_date =
-      this.props.maximum_date != null ? this.props.maximum_date : "2020-06-23";
+    const initDate =
+      this.props.initialDate != null ? this.props.initialDate : "2020-02-20";
+    const maximDate =
+      this.props.maximumDate != null ? this.props.maximumDate : "2020-06-23";
 
     // converting DateFields that come from the DatePicker to string. If there is no
     // counterfactual date use the default historical one
-    const counterfactual_first_restrictions_date =
-      this.props.counterfactual_first_restrictions_date != null
-        ? convert(this.props.counterfactual_first_restrictions_date)
-        : this.props.first_restrictions_date;
-    const counterfactual_lockdown_date =
-      this.props.counterfactual_lockdown_date != null
-        ? convert(this.props.counterfactual_lockdown_date)
-        : this.props.lockdown_date;
+    const counterfactualFirstRestrictions =
+      this.props.counterfactualFirstRestrictionsDate != null
+        ? convert(this.props.counterfactualFirstRestrictionsDate)
+        : this.props.firstRestrictionsDate;
+    const counterfactualLockdownDate =
```

Comment:
Similarly here: maybe better as `counterfactualDateLockdown` and `counterfactualDateFirstRestrictions`?

Reply:
Done